poc_iot_verifier/src/runner.rs

```diff
@@ -161,7 +161,7 @@ impl Runner {
 
             let db_witnesses = Report::get_witnesses_for_beacon(&self.pool, packet_data).await?;
             let witness_len = db_witnesses.len();
-            tracing::info!("found {witness_len} witness for beacon");
+            tracing::debug!("found {witness_len} witness for beacon");
```

Comment:
```suggestion
            tracing::debug!("found {witness_len} witness for beacon");
            metrics::gauge!("oracles_poc_iot_verifier_witnesses_per_beacon", witness_len);
```

Reply:
I'll later add a global workspace and crate name prefixer thing, like in `file_sink_write!`, so it can become the less typo-prone `gauge!("witnesses_per_beacon", witness_len);`.